platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/internal/ConsensusRound.java

```diff
@@ -47,17 +47,23 @@ public class ConsensusRound implements Round {
     /** The number of application transactions in this round */
     private int numAppTransactions = 0;
 
+    /** The minimum generation of the non-ancient events in this round */
+    private final long minimumGenerationNonAncient;
+
     /**
      * Create a new instance with the provided consensus events.
      *
-     * @param consensusEvents
-     * 		the events in the round, in consensus order
-     * @param generations
-     * 		the consensus generations for this round
+     * @param consensusEvents             the events in the round, in consensus order
+     * @param generations                 the consensus generations for this round
+     * @param minimumGenerationNonAncient the minimum generation of the non-ancient events in this round
      */
-    public ConsensusRound(final List<EventImpl> consensusEvents, final GraphGenerations generations) {
+    public ConsensusRound(
+            final List<EventImpl> consensusEvents,
+            final GraphGenerations generations,
+            final long minimumGenerationNonAncient) {
```

Comment:
again, this value is a duplicate of the one contained in generations

Reply:
removed